### ISSUE TITLE
Failed to load ComfyUI workflow editor

### ISSUE THREAD

**Issue by adrien731** [Stability-AI/StableSwarmUI#280]
After installing StableSwarmUI I had this problem with Failed to load ComfyUI workflow editor. Although I followed the correct steps to install StableSwarmUI.

**Comment by mcmonkey4eva** [comment]
Go to Server -> Logs and check for errors. If it's not obvious from there, swap "View type" in the logs page to "Debug".
Most likely your comfy instance failed to load, the reason should be identifiable from logs. If you can't figure it out yourself from the logs, copy/paste the log text here

**Comment by adrien731** [comment]
> انتقل إلى الخادم -> السجلات وتحقق من الأخطاء. إذا لم يكن الأمر واضحًا من هناك، فقم بتبديل "نوع العرض" في صفحة السجلات إلى "تصحيح الأخطاء". على الأرجح فشل تحميل مثيلك المريح، ويجب تحديد السبب من خلال السجلات. إذا لم تتمكن من معرفة ذلك بنفسك من السجلات، فانسخ/ألصق نص السجل هنا

Thank you. I followed the procedures you mentioned, but unfortunately I did not discover the problem. Here is the text of the log.  


2024-03-24 01:58:48.372 [Init] === StableSwarmUI v0.6.2.0 Starting ===
2024-03-24 01:58:51.015 [Init] Prepping extension: StableSwarmUI.Builtin_StabilityAPIExtension.StabilityAPIExtension...
2024-03-24 01:58:51.369 [Init] Prepping extension: StableSwarmUI.Builtin_ScorersExtension.ScorersExtension...
2024-03-24 01:58:51.386 [Init] Prepping extension: StableSwarmUI.Builtin_ImageBatchToolExtension.ImageBatchToolExtension...
2024-03-24 01:58:51.396 [Init] Prepping extension: StableSwarmUI.Builtin_GridGeneratorExtension.GridGeneratorExtension...
2024-03-24 01:58:51.418 [Init] Prepping extension: StableSwarmUI.Builtin_DynamicThresholding.DynamicThresholdingExtension...
2024-03-24 01:58:51.432 [Init] Prepping extension: StableSwarmUI.Builtin_ComfyUIBackend.ComfyUIBackendExtension...
2024-03-24 01:58:51.456 [Init] Prepping extension: StableSwarmUI.Builtin_AutoWebUIExtension.AutoWebUIBackendExtension...
2024-03-24 01:58:52.974 [Init] Parsing command line...
2024-03-24 01:58:52.977 [Init] Loading settings file...
2024-03-24 01:58:53.382 [Init] Re-saving settings file...
2024-03-24 01:58:53.508 [Init] Applying command line settings...
2024-03-24 01:58:53.811 [Debug] [Load Time] Initial settings load took 5.28s (5.28s from start)
2024-03-24 01:58:55.088 [Debug] [Load Time] Extension PreInit took 1.28s (6.56s from start)
2024-03-24 01:58:55.088 [Init] Prepping options...
2024-03-24 01:58:58.165 [Debug] [Load Time] Prep Objects took 3.06s (9.63s from start)
2024-03-24 01:58:58.165 [Debug] [Load Time] Web PreInit took 0.02s (9.64s from start)
2024-03-24 01:58:58.338 [Debug] [Load Time] Extensions Init took 0.17s (9.81s from start)
2024-03-24 01:58:58.339 [Debug] [Load Time] Prep Utils took 0s (9.81s from start)
2024-03-24 01:58:59.625 [Debug] [Load Time] Languages load took 1.28s (11.09s from start)
2024-03-24 01:58:59.625 [Init] Loading models list...
2024-03-24 01:59:01.408 [Debug] [Load Time] Model listing took 1.78s (12.88s from start)
2024-03-24 01:59:01.408 [Init] Loading backends...
2024-03-24 01:59:01.440 [Init] Loading backends from file...
2024-03-24 01:59:01.690 [Init] Initializing backend #0 - ComfyUI Self-Starting...
2024-03-24 01:59:01.742 [Init] Backend request handler loop ready...
2024-03-24 01:59:01.748 [Debug] [Load Time] Backends took 0.34s (13.22s from start)
2024-03-24 01:59:01.748 [Init] Prepping API...
2024-03-24 01:59:01.767 [Debug] [Load Time] API took 0.02s (13.23s from start)
2024-03-24 01:59:01.767 [Init] Prepping webserver...
2024-03-24 01:59:11.684 [Debug] [Load Time] [Web] WebApp builder prep took 7.72s (7.72s from start)
2024-03-24 01:59:18.954 [Debug] ComfyUI (Git Pull) stderr: From https://github.com/comfyanonymous/ComfyUI
2024-03-24 01:59:18.954 [Debug] ComfyUI (Git Pull) stderr:    327ca13..5d8898c  master     -> origin/master
2024-03-24 01:59:19.691 [Debug] [Load Time] [Web] WebApp build took 8s (15.72s from start)
2024-03-24 01:59:19.982 [Debug] [Load Time] [Web] exception handler took 0.3s (16.02s from start)
2024-03-24 01:59:19.982 [Debug] [Load Time] [Web] StartStop handler took 0s (16.02s from start)
2024-03-24 01:59:20.142 [Debug] [Load Time] [Web] static files took 0.16s (16.17s from start)
2024-03-24 01:59:23.593 [Debug] [Load Time] [Web] core use calls took 3.45s (19.63s from start)
2024-03-24 01:59:24.193 [Debug] ComfyUI (Git Pull) stdout: Updating 327ca13..5d8898c
2024-03-24 01:59:24.193 [Debug] ComfyUI (Git Pull) stdout: Fast-forward
2024-03-24 01:59:24.232 [Debug] [Load Time] [Web] core maps took 0.64s (20.27s from start)
2024-03-24 01:59:24.232 [Init] Scan for web extensions...
2024-03-24 01:59:24.575 [Debug] ComfyUI (Git Pull) stdout:  comfy/model_management.py | 16 +++++++++++-----
2024-03-24 01:59:24.575 [Debug] ComfyUI (Git Pull) stdout:  execution.py              |  1 +
2024-03-24 01:59:24.575 [Debug] ComfyUI (Git Pull) stdout:  2 files changed, 12 insertions(+), 5 deletions(-)
2024-03-24 01:59:24.591 [Info] Self-Start ComfyUI (Git Pull) exited properly.
2024-03-24 01:59:24.595 [Debug] Status of ComfyUI (Git Pull) after process end is RUNNING
2024-03-24 01:59:24.829 [Debug] Requested generic launch of dlbackend/comfy/ComfyUI/main.py on GPU 0 from ComfyUI-0
2024-03-24 01:59:24.836 [Debug] (ComfyUI-0 launch) Adding path C:\111\Voic\StableSwarmUI\dlbackend\comfy\python_embeded
2024-03-24 01:59:24.836 [Debug] (ComfyUI-0 launch) Will use python: C:\111\Voic\StableSwarmUI\dlbackend\comfy\python_embeded\python.exe
2024-03-24 01:59:24.958 [Init] Self-Start ComfyUI-0 on port 7821 is loading...
2024-03-24 01:59:25.017 [Debug] [Load Time] [Web] end took 0.78s (21.05s from start)
2024-03-24 01:59:25.017 [Debug] [Load Time] Web prep took 23.25s (36.48s from start)
2024-03-24 01:59:25.017 [Init] Readying extensions for launch...
2024-03-24 01:59:25.023 [Debug] [Load Time] Extensions pre-launch took 0.02s (36.5s from start)
2024-03-24 01:59:25.023 [Init] Launching server...
2024-03-24 01:59:25.026 [Init] Starting webserver on http://localhost:7801
2024-03-24 01:59:31.391 [Debug] [Load Time] Web launch took 6.36s (42.86s from start)
2024-03-24 01:59:31.391 [Init] Program is running.
2024-03-24 01:59:31.950 [Init] Launch web browser...
2024-03-24 01:59:44.956 [Info] Creating new admin session 'local' for ::1
2024-03-24 02:00:12.776 [Debug] ComfyUI-0 port 7821 waiting for server...
2024-03-24 02:01:03.645 [Debug] ComfyUI-0 port 7821 waiting for server...
2024-03-24 02:01:54.495 [Debug] ComfyUI-0 port 7821 waiting for server...
2024-03-24 02:02:45.392 [Debug] ComfyUI-0 port 7821 waiting for server...
2024-03-24 02:03:36.313 [Debug] ComfyUI-0 port 7821 waiting for server...
2024-03-24 02:04:04.047 [Warning] ComfyUI-0 on port 7821 stderr: Traceback (most recent call last):
2024-03-24 02:04:04.052 [Debug] ComfyUI-0 on port 7821 stderr:   File "C:\111\Voic\StableSwarmUI\dlbackend\comfy\ComfyUI\main.py", line 76, in <module>
2024-03-24 02:04:04.090 [Debug] ComfyUI-0 on port 7821 stderr:     import execution
2024-03-24 02:04:04.104 [Debug] ComfyUI-0 on port 7821 stderr:   File "C:\111\Voic\StableSwarmUI\dlbackend\comfy\ComfyUI\execution.py", line 11, in <module>
2024-03-24 02:04:04.105 [Debug] ComfyUI-0 on port 7821 stderr:     import nodes
2024-03-24 02:04:04.105 [Debug] ComfyUI-0 on port 7821 stderr:   File "C:\111\Voic\StableSwarmUI\dlbackend\comfy\ComfyUI\nodes.py", line 21, in <module>
2024-03-24 02:04:04.162 [Debug] ComfyUI-0 on port 7821 stderr:     import comfy.diffusers_load
2024-03-24 02:04:04.162 [Debug] ComfyUI-0 on port 7821 stderr:   File "C:\111\Voic\StableSwarmUI\dlbackend\comfy\ComfyUI\comfy\diffusers_load.py", line 3, in <module>
2024-03-24 02:04:04.214 [Debug] ComfyUI-0 on port 7821 stderr:     import comfy.sd
2024-03-24 02:04:04.214 [Debug] ComfyUI-0 on port 7821 stderr:   File "C:\111\Voic\StableSwarmUI\dlbackend\comfy\ComfyUI\comfy\sd.py", line 5, in <module>
2024-03-24 02:04:04.311 [Debug] ComfyUI-0 on port 7821 stderr:     from comfy import model_management
2024-03-24 02:04:04.311 [Debug] ComfyUI-0 on port 7821 stderr:   File "C:\111\Voic\StableSwarmUI\dlbackend\comfy\ComfyUI\comfy\model_management.py", line 38, in <module>
2024-03-24 02:04:04.312 [Debug] ComfyUI-0 on port 7821 stderr:     import torch_directml
2024-03-24 02:04:04.312 [Debug] ComfyUI-0 on port 7821 stderr: ModuleNotFoundError: No module named 'torch_directml'
2024-03-24 02:04:14.607 [Info] Self-Start ComfyUI-0 on port 7821 unexpectedly exited (if something failed, change setting `LogLevel` to `Debug` to see why!)
2024-03-24 02:04:14.610 [Debug] Status of ComfyUI-0 on port 7821 after process end is LOADING
2024-03-24 02:04:14.610 [Info] Self-Start ComfyUI-0 on port 7821 had errors before shutdown:
ComfyUI-0 on port 7821 error: Traceback (most recent call last):
ComfyUI-0 on port 7821 error:   File "C:\111\Voic\StableSwarmUI\dlbackend\comfy\ComfyUI\main.py", line 76, in <module>
ComfyUI-0 on port 7821 error:     import execution
ComfyUI-0 on port 7821 error:   File "C:\111\Voic\StableSwarmUI\dlbackend\comfy\ComfyUI\execution.py", line 11, in <module>
ComfyUI-0 on port 7821 error:     import nodes
ComfyUI-0 on port 7821 error:   File "C:\111\Voic\StableSwarmUI\dlbackend\comfy\ComfyUI\nodes.py", line 21, in <module>
ComfyUI-0 on port 7821 error:     import comfy.diffusers_load
ComfyUI-0 on port 7821 error:   File "C:\111\Voic\StableSwarmUI\dlbackend\comfy\ComfyUI\comfy\diffusers_load.py", line 3, in <module>
ComfyUI-0 on port 7821 error:     import comfy.sd
ComfyUI-0 on port 7821 error:   File "C:\111\Voic\StableSwarmUI\dlbackend\comfy\ComfyUI\comfy\sd.py", line 5, in <module>
ComfyUI-0 on port 7821 error:     from comfy import model_management
ComfyUI-0 on port 7821 error:   File "C:\111\Voic\StableSwarmUI\dlbackend\comfy\ComfyUI\comfy\model_management.py", line 38, in <module>
ComfyUI-0 on port 7821 error:     import torch_directml
ComfyUI-0 on port 7821 error: ModuleNotFoundError: No module named 'torch_directml'
2024-03-24 02:04:15.964 [Debug] ComfyUI-0 self-start port 7821 loop ending (should now be alive)
2024-03-24 02:04:16.330 [Debug] Data refreshed!

**Comment by mcmonkey4eva** [comment]
Ah, you're on an AMD GPU? See here https://github.com/Stability-AI/StableSwarmUI/issues/23
tldr: AMD GPUs are a pain to get working properly, but it can be done with a bit of manual forcing it

**Comment by adrien731** [comment]
Thank you, the problem has been solved

**Comment by dapa5900** [comment]
Hey. My workflow tab isn't loading as well. I'm on a 4090. Here's the debug log. Thanks!

2024-08-16 15:35:33.769 [Debug] ComfyBackendDirect call was misrouted, rerouting to '/ComfyBackendDirect/user.css'
2024-08-16 15:35:33.769 [Debug] ComfyBackendDirect call was misrouted, rerouting to '/ComfyBackendDirect/materialdesignicons.min.css'
2024-08-16 15:35:33.769 [Debug] ComfyBackendDirect call was misrouted, rerouting to '/ComfyBackendDirect/assets/index-DIiqwEjy.js'
2024-08-16 15:35:33.769 [Debug] ComfyBackendDirect call was misrouted, rerouting to '/ComfyBackendDirect/assets/index-CwWW6Xjy.css'
2024-08-16 15:35:33.784 [Debug] [Load ComfyUI Self-Starting #0] Got valid value set, will parse...
2024-08-16 15:35:33.784 [Debug] Comfy backend 0 using model folder format: backslash \ due to model OfficialStableDiffusion\sd_xl_base_1.0.safetensors
2024-08-16 15:35:33.784 [Debug] [Load ComfyUI Self-Starting #0] Done parsing value set.
2024-08-16 15:35:33.787 [Debug] Data refreshed!
2024-08-16 15:35:34.175 [Debug] ComfyUI redirection gave non-200 code: '404' for URL: GET 'api/userdata/workflows%2F.index.json'
2024-08-16 15:35:34.186 [Debug] ComfyBackendDirect call was misrouted, rerouting to '/ComfyBackendDirect/assets/index-DjWyclij.css'

**Comment by mcmonkey4eva** [comment]
@dapa5900 this is the old Stable repo, see modern independent repo @ https://github.com/mcmonkeyprojects/SwarmUI/issues/169#issuecomment-2294346528

**Comment by Zheiden** [comment]
23:39:49.357 [Init] === SwarmUI v0.9.6.2 Starting at 2025-06-19 23:39:49 ===
23:39:49.421 [Init] Prepping extension: SwarmUI.Builtin_ScorersExtension.ScorersExtension...
23:39:49.422 [Init] Prepping extension: SwarmUI.Builtin_ImageBatchToolExtension.ImageBatchToolExtension...
23:39:49.423 [Init] Prepping extension: SwarmUI.Builtin_GridGeneratorExtension.GridGeneratorExtension...
23:39:49.424 [Init] Prepping extension: SwarmUI.Builtin_DynamicThresholding.DynamicThresholdingExtension...
23:39:49.425 [Init] Prepping extension: SwarmUI.Builtin_ComfyUIBackend.ComfyUIBackendExtension...
23:39:49.425 [Init] Prepping extension: SwarmUI.Builtin_AutoWebUIExtension.AutoWebUIBackendExtension...
23:39:49.481 [Init] Parsing command line...
23:39:49.482 [Init] Loading settings file...
23:39:49.511 [Init] Re-saving settings file...
23:39:49.546 [Init] Applying command line settings...
23:39:49.566 [Init] SwarmUI was installed 2025-06-19 (0 days ago) with version 0.9.6.2
23:39:49.613 [Init] Prepping options...
23:39:49.646 [Init] Current git commit is [a0dae3b7: minor fix on Refine Image button], marked as date 2025-06-19 19:08:27 (1 hour ago)
23:39:49.784 [Init] CPU Cores: 4 | RAM: 15.97 GiB total, 9.90 GiB available, 18.72 GiB total page file, 9.55 GiB available page file
23:39:49.799 [Init] GPU 0: NVIDIA GeForce GTX 1070 | Temp 33C | Util 1% GPU, 2% Memory | VRAM 8.00 GiB total, 7.27 GiB free, 621.00 MiB used
23:39:50.166 [Init] Loading models list...
23:39:50.320 [Init] Loading backends...
23:39:50.323 [Init] Loading backends from file...
23:39:50.346 [Init] Prepping API...
23:39:50.346 [Init] Initializing backend #0 - ComfyUI Self-Starting...
23:39:50.350 [Init] Prepping webserver...
23:39:50.355 [Init] Backend request handler loop ready...
23:39:50.737 [Init] Scan for web extensions...
23:39:50.741 [Init] Swarm is up to date! You have version 0.9.6.2, and 0.9.6-Beta is the latest.
23:39:50.836 [Init] Readying extensions for launch...
23:39:50.837 [Init] Launching server...
23:39:50.839 [Init] Starting webserver on http://localhost:7801
23:39:50.904 [Init] SwarmUI v0.9.6.2 - Local is now running.
23:39:51.411 [Init] Launch web browser...
23:39:51.659 [Init] Self-Start ComfyUI-0 on port 7821 is loading...
23:39:52.755 [Info] Creating new session 'local' for ::1
23:39:53.974 [Error] Self-Start ComfyUI-0 on port 7821 failed to start: Server returned entirely empty response, something went wrong.
